.circleci/config.yml

```
@@ -201,10 +201,18 @@ jobs:
           command: docker load -i /tmp/workspace/merinopy.tar.gz
       - dockerhub-login
       - run:
-          name: Push to Docker Hub
+          name: Push to Docker Hub # Load tests are triggered though [load test: (abort|warn)] label.
           command: |
             if [ "${CIRCLE_BRANCH}" == "main" ]; then
               DOCKER_TAG="${CIRCLE_SHA1}"
+              if git log -1 "$CIRCLE_SHA1" | grep -q '\[load test: warn\]'; then
+                  echo "Load test requested. Slack warning will be output if test fails and deployment workflow for prod will proceed."
+                  DOCKER_TAG="stage-loadtest-warn-${CIRCLE_SHA1}"
+              fi
+              if git log -1 "$CIRCLE_SHA1" | grep -q '\[load test: abort\]'; then
+                  echo "Load test requested. Deployment workflow for prod will abort if load test fails."
+                  DOCKER_TAG="stage-loadtest-abort-${CIRCLE_SHA1}"
+              fi
```

We will need to add the `stage-` prefix to all stage Docker tags. So that line 207 becomes `DOCKER_TAG="stage-${CIRCLE_SHA1}"`.

Going one step further, we could improve evaluation time by using elif and else statements.

```suggestion
    if git log -1 "$CIRCLE_SHA1" | grep -q '\[load test: warn\]'; then
        echo "Load test requested. Slack warning will be output if test fails and deployment workflow for prod will proceed."
        DOCKER_TAG="stage-loadtest-warn-${CIRCLE_SHA1}"
    elif git log -1 "$CIRCLE_SHA1" | grep -q '\[load test: abort\]'; then
        echo "Load test requested. Deployment workflow for prod will abort if load test fails."
        DOCKER_TAG="stage-loadtest-abort-${CIRCLE_SHA1}"
    else
        DOCKER_TAG="stage-${CIRCLE_SHA1}"
    fi
```